sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/Models/TimeSeries2Id.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Iot.TimeSeriesInsights
+{
+    /// <summary>
+    /// A single Time Series Id value that is an array of 2 string values that uniquely identifies a Time Series instance.
+    /// </summary>
```

Comment:
Can add para here and in other classes

"TSI service allows to provide three key properties as your Time Series Id when creating environment. Their combination will be a composite key that represents the Time Series Id.

Use this class if you chose two properties as your Time Series Id while creating Time Series Insights environment."